<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emcDialog</class>
 <widget class="QDialog" name="emcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ear Masa Cob</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="logoWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="logoLayout">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Square Array</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QgsMapLayerComboBox" name="arrayComboBox">
           <property name="allowEmptyLayer">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="arrayButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Spectral Library File (in case on a different location)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsFileWidget" name="browseCSV">
         <property name="toolTip">
          <string>Browse for the metadata</string>
         </property>
         <property name="dialogTitle">
          <string>Select a metadata file</string>
         </property>
         <property name="filter">
          <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="classLabel">
         <property name="text">
          <string>Metadata element with endmember classes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="classDropDown">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Choose the metadata element that contains the classification used in MESMA</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Output Spectral Library</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsFileWidget" name="browseOut">
         <property name="toolTip">
          <string>Choose a name and location for the EMC output (.csv).</string>
         </property>
         <property name="dialogTitle">
          <string>Select a metadat file (output)</string>
         </property>
         <property name="filter">
          <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_log">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="logBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="OKClose">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="arrayAction">
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconText">
    <string>...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
